determined-ai/determined
# fix: storybook rendering issue

## Description

[DET-7759]



## Test Plan

1. Run storybook to see if it works. I tested with `npm run storybook`
2. Check the theme (Light mode or Dark mode)



## Commentary (optional)

~~On my environment, theme is always dark mode. Not sure if someone else gets the same issue or not. 
The background color is white, so I cannot see white text in dark mode.~~ 
Theme on Storybook depends on system mode

Also, I added `StoreProvider` and `BrowserRouter` in the top level because I saw some errors that come from them. Instead of adding these in each story, I thought adding these in the top level is a good idea to avoid the errors in the future. For example, `context`  related hooks would be used in some components but forget to add `decorators` in the story etc...




## Checklist

See [Release Note](../docs/release-notes/README.md) for details.





[DET-7759]: https://determinedai.atlassian.net/browse/DET-7759?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
